cmd/controllers/app/server.go

```diff
@@ -85,11 +86,13 @@ func Run(opt *options.ServerOption) error {
 	jobController := job.NewJobController(kubeClient, kbClient, vkClient)
 	queueController := queue.NewQueueController(kubeClient, kbClient)
 	garbageCollector := garbagecollector.New(vkClient)
+	pgController := podgroup.NewPodgroupController(kubeClient, kbClient, vkClient)
 
 	run := func(ctx context.Context) {
 		go jobController.Run(ctx.Done())
 		go queueController.Run(ctx.Done())
 		go garbageCollector.Run(ctx.Done())
+		go pgController.Run(ctx.Done())
```

Comment:
We need make this controller running configurable, since in most of the volcano only cases, this is not needed.

Reply:
add normal-job-pg-ctrl, normally fasle, disable pgcontroller